.github/workflows/withdraw-packages.yaml

```diff
@@ -0,0 +1,50 @@
+name: Withdraw packages
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Withdraw packages
+    runs-on: ubuntu-latest
+
+    permissions:
+      packages: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: chainguard-dev/actions/setup-melange@main
+
+      - uses: google-github-actions/setup-gcloud@v0
+
+      - run: echo "${{ secrets.MELANGE_RSA }}" > ./wolfi-signing.rsa
+      - run: |
+          sudo mkdir -p /etc/apk/keys
+          sudo cp ./wolfi-signing.rsa.pub /etc/apk/keys/wolfi-signing.rsa.pub
+
+      - name: 'Delete withdrawn packages'
+        run: |
+          for arch in x86_64 aarch64; do
+            for pkg in $(cat withdrawn-packages.txt); do
```

Comment:
FR: ignore empty lines and lines starting with # so we can include comments about why they're withdrawn, links to issues, etc.